app/view.py

```diff
@@ -237,6 +237,7 @@ def post(self):
                 confirmed=False)
             # Write the new NoteWithBadWords to the datastore
             db.put(note)
+            UsageCounter.increment_note_counter(self.repo)
```

Comment:
This code may no longer be called. Can you confirm that add_note.py is used instead of this file when you add note? If so, can you add a TODO comment to remove this method? In that case, you can leave this file unchanged other than adding a TODO comment.